docs/main_text.md

```diff
@@ -214,5 +214,17 @@ TODO: screenshots? plots that show effect of different scaling options?
 
 # Species compendia
 
+TODO: Note about release schedule and/or presence on Zenodo
+
+refine.bio periodically releases compendia comprised of all samples from a species that were run on one of the platforms we support.
```

Comment:
This reads like it could also be per-platform. What about:
> refine.bio periodically releases compendia comprised of all the samples from a species that we were able to process.